region/lowernorfair/west.json

```diff
@@ -636,6 +668,29 @@
                     "It actually takes 29 supers but giving 1 extra in leniency since it's easy to miss"
                   ]
                 },
+                {
+                  "name": "Golden Torizo Missiles Only",
+                  "notable": true,
+                  "requires": [
+                    "h_heatProof",
+                    "Morph",
+                    "Missile",
```

Comment:
In theory the fight takes just as long with 5 missile capacity because you can only shoot 2 at a time and return to farming position after every motion.  But having more does speed up the fight a meaningful amount.